tests/testthat/test-temp-tac.R

```diff
@@ -0,0 +1,42 @@
+#
+# will move this to test-histogram.R when finished with this issue/PR
+#
+
+test_that("histogram.dt does not produce corrupted bins when given TAC data from issue 182", {
+  map <- data.frame('id' = c('values'),
+                    'plotRef' = c('xAxisVariable'),
+                    'dataType' = c('NUMBER'),
+                    'dataShape' = c('CONTINUOUS'),
+                    stringsAsFactors = FALSE)
+
+  dt <- histogram.dt(data.issue182, map, binWidth = 1, value = 'count', barmode = 'stack', binReportValue = 'binWidth')
```

Comment:
is there a reasonable way we can modify the existing test object (like in line 25) to fail in the same way? Ideally we've been trying to keep the test data to a minimum set that can be used or manipulated for nearly all the tests. Or if we could pick out 500 values of the tac data and add them as a new column `contE` or something like that in testDF that would similarly prevent test data object sprawl, if possible.

Reply:
We'd need to make sure that the new test data column contE fails the new test with the old code. That's making my head explode at the moment...

I suppose we could make a new branch from `main`, add the data the new way as `testDF.contE`, and copy over the new test, and make sure it fails.  Then merge that branch into this one.  Yeah, that should work.

Reply:
I've made a [new ticket](https://github.com/VEuPathDB/plot.data/issues/188) for this and the function move to veupathUtils.